gluten-ut/spark32/src/test/scala/org/apache/spark/sql/GlutenDateFunctionsSuite.scala

```diff
@@ -234,10 +221,8 @@ class GlutenDateFunctionsSuite extends DateFunctionsSuite with GlutenSQLTestsTra
           if (legacyParserPolicy == "legacy") {
             // In Spark 2.4 and earlier, to_timestamp() parses in seconds precision and cuts off
             // the fractional part of seconds. The behavior was changed by SPARK-27438.
-            // Ignore this test case. Velox returns null for such case.
-            // val legacyFmt = "yyyy/MM/dd HH:mm:ss"
-            // checkAnswer(df.select(to_timestamp(col("s"), legacyFmt)), Seq(
-            //  Row(ts1), Row(ts2)))
+            val legacyFmt = "yyyy/MM/dd HH:mm:ss"
+            checkAnswer(df.select(to_timestamp(col("s"), legacyFmt)), Seq(Row(ts1), Row(ts2)))
```

Comment:
You can remove testGluten "to_timestamp" and enable vanilla Spark test "to_timestamp" because they are already the same.